scripts/image_vars.src

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-# This file is autogenerated -- update and re-run ./scripts/image-names.sh instead.
```

Comment:
should this comment change be reverted to restore the `./scripts/` reference?

Reply:
Fixed